tests/test_typing.py

```diff
@@ -160,13 +146,17 @@ def test_func_ns_excludes_default_globals() -> None:
         assert default_global_var not in func_ns
 
 
-module_foo = 'global_foo'
-module_ns = parent_frame_namespace(parent_depth=1)
+def test_module_ns_is_none(create_module) -> None:
```

Comment:
Can we add a test with `force=True`, and guarantee that the result is not `None` as well?